sound_manager.py

```diff
@@ -0,0 +1,81 @@
+from arcade import Sound, load_sound
+
+DEFAULT_VOLUME = 0.7
+SFX_BEETLE_HIT = "Assets/Sound/SFX/impact2_noisecollector.wav"
+SFX_BEETLE_DEAD = "Assets/Sound/SFX/beetle_dead_mixkit.wav"
+SFX_PEASHOOTER = "Assets/Sound/SFX/pew_xtrgamr.wav"
+
+MUSIC_BANJOS_PLACEHOLDER = "Assets/Sound/Music/banjo_loop.wav"
+
+BEETLE_HIT = "BeetleHit"
+BEETLE_DEAD = "BeetleDead"
+PEASHOOTER_PEW = "PeashooterPew"
+BANJOS = "Banjos"
+
+class SoundManager:
+    # This creates the sound manager as a singleton instance
+    def __new__(cls):
+        if not hasattr(cls, 'instance'):
```

Comment:
Ooh, little nitpick I just noticed: We should use double quotes here instead of single quotes, as per our general style elsewhere.